reference/api/search.mdx

```diff
@@ -136,7 +136,7 @@ This endpoint only accepts [string filter expressions](/learn/advanced/filtering
 This endpoint should only be used when no API key is required. If an API key is required, use the [POST](/reference/api/search#search-in-an-index-with-post-route) route instead.
 
 <Capsule intent="note">
-By default, [this endpoint returns a maximum of 1000 results](/learn/advanced/known_limitations#maximum-number-of-results-per-search). If you want to scrape your database, use the [get documents endpoint](/reference/api/documents#get-documents) instead.
+By default, [this endpoint returns a maximum of 1000 results](/learn/advanced/known_limitations#maximum-number-of-results-per-search). If you want to scrape your database, use the [get documents endpoint](/reference/api/documents#get-documents-with-post) instead.
```

Comment:
Do we need to set up redirects for `/api/documents#get-documents`?

Reply:
Good catch. I always forget the redirects 😞